iree/turbine/kernel/wave/templates/quantized_attention.py

```diff
@@ -187,7 +186,7 @@ def repeat(
         # repeat represents the results of the loop
         res_max, res_sum, res_mm = repeat
         reciprocal_sum = tkw.reciprocal(res_sum)
-        res = res_mm * reciprocal_sum * v_dequant * fp8_dequant
```

Comment:
we can remove the fp8_dequant because we are now applying /fp8_offset into e_delta -> d_j (denominator) as well as e_delta -> mma(numerator) => we no longer need to do `fp8_dequant`.